src/test/java/com/github/wovnio/wovnjava/HeadersTest.java

```diff
@@ -240,24 +240,60 @@ public void testLocationWithSitePrefixPath() throws ConfigurationError {
 
     public void testGetIsValidRequest() throws ConfigurationError {
         Headers h;
-        h = makeHeaderWithSitePrefixPath("/", "global");
+        h = createHeaders("/", "global", "");
         assertEquals(false, h.getIsValidRequest());
 
-        h = makeHeaderWithSitePrefixPath("/global", "global");
+        h = createHeaders("/global", "global", "");
         assertEquals(true, h.getIsValidRequest());
 
-        h = makeHeaderWithSitePrefixPath("/global/ja/foo", "global");
+        h = createHeaders("/global/ja/foo", "global", "");
         assertEquals(true, h.getIsValidRequest());
 
-        h = makeHeaderWithSitePrefixPath("/ja/global/foo", "global");
+        h = createHeaders("/ja/global/foo", "global", "");
+        assertEquals(false, h.getIsValidRequest());
+
+
+        h = createHeaders("/", "", "/admin,/wp-admin");
```

Comment:
move the ignore class test cases into a separate test: `testGetIsValidRequest__withIgnoredPaths`